sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java

```diff
@@ -101,6 +101,7 @@ static MetricExporter configureOtlpMetrics(
           builder::setTimeout,
           builder::setTrustedCertificates,
           (unused) -> {});
+      OtlpConfigUtil.configureOtlpAggregationTemporality(config, builder::setPreferredTemporality);
```

Comment:
Why treat this differently than the other setters? Are you opposed to adding another chunk to the monster method? I'm just thinking about consistency.

Reply:
Because this only applies to the metric exporters. The other method sets the common properties for span and metric exporters.